draft-ietf-ipsecme-rfc8229bis.xml

```diff
@@ -683,26 +706,35 @@
           supports NAT traversal. However, for the transport mode IPsec SAs, implementations 
           need to handle TCP and UDP packet checksum fixup during decapsulation, 
           as defined for UDP encapsulation in <xref target="RFC3948"/>.
-          Implementations MAY use the information that
+          </t>
+
+          <t>Implementations MAY use the information that
           a NAT is present to influence keep-alive timer values.
           </t>
 
         </section>
 
-        <section title="Keep-Alives and Dead Peer Detection" anchor="dpd-nat-ka">
-          <t>Encapsulating IKE and IPsec inside of a TCP connection can impact the
-          strategy that implementations use to detect peer liveness and to
-          maintain middlebox port mappings.  Peer liveness should be checked
-          using IKE informational packets <xref target="RFC7296"/>.
-          </t>
+        <section title="NAT Keep-Alives Packets" anchor="nat-ka">
```

Comment:
```suggestion
        <section title="NAT Keep-Alives Packet" anchor="nat-ka">
```

Reply:
I checked RFC 3948 and it uses term NAT-keepalive packet. I think we should use the same spelling for consistency.

Reply:
Oh, sorry I did the wrong suggestion! It was to not be "keep-alives"